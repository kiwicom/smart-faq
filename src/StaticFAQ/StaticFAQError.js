// @flow

import * as React from 'react';
import css from 'styled-jsx/css';
import { Text, Heading } from '@kiwicom/orbit-components';
import Trans from '@kiwicom/nitro/lib/components/Text';

import { ContactPageLink } from '../common';

const style = css`
  .staticFAQError {
    height: 100%;
    display: flex;
    justify-content: center;
    flex-direction: column;
  }
  .title {
    margin-bottom: 8.4px;
  }
  div.contactLink {
    margin-top: 16.8px;
  }
`;

const StaticFAQError = () => (
  <div className="staticFAQError">
    <div className="title">
      <Heading>
        <Trans t={__('smartfaq.single_booking_page.booking_error.title')} />
      </Heading>
    </div>
    <Text type="secondary" size="large">
      <Trans t={__('smartfaq.faq.error_page.description')} />
    </Text>
    <div className="contactLink">
      <ContactPageLink />
    </div>
    <style jsx>{style}</style>
  </div>
);

export default StaticFAQError;
