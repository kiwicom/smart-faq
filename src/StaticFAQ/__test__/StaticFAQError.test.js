// @flow

import * as React from 'react';
import { render } from 'enzyme';

import StaticFAQError from '../StaticFAQError';

describe('StaticFAQError', () => {
  it('should match snapshot', () => {
    const result = render(<StaticFAQError />);
    expect(result).toMatchSnapshot();
  });
});
