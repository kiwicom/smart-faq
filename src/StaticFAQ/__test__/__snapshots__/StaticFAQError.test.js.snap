// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StaticFAQError should match snapshot 1`] = `
<div
  class="jsx-606556222 staticFAQError"
>
  <div
    class="jsx-606556222 title"
  >
    <h1
      class="Heading__StyledHeading-sALAQ gfkQAC"
    >
      smartfaq.single_booking_page.booking_error.title
    </h1>
  </div>
  <p
    class="Text__StyledText-inVtPV gJJlTV"
  >
    smartfaq.faq.error_page.description
  </p>
  <div
    class="jsx-606556222 contactLink"
  >
    <div
      class="jsx-3405637160 contactUs"
    >
      <a
        class="jsx-3405637160"
        href="/en/content/feedback"
        target="_blank"
      >
        smartfaq.contact_page_link.default
        <span
          class="jsx-3405637160 open-icon"
        >
          <svg
            fill="currentColor"
            height="16px"
            preserveAspectRatio="xMidYMid meet"
            style="vertical-align:middle;color:#171b1e"
            viewBox="0 0 24 24"
            width="16px"
          >
            <path
              d="M17.444 17.444H6.556V6.556H12V5H6.556C5.692 5 5 5.7 5 6.556v10.888C5 18.3 5.692 19 6.556 19h10.888C18.3 19 19 18.3 19 17.444V12h-1.556v5.444zM13.556 5v1.556h2.792L8.702 14.2 9.8 15.298l7.645-7.646v2.792H19V5h-5.444z"
            />
          </svg>
        </span>
      </a>
    </div>
  </div>
</div>
`;
